cce/cce.go

```diff
@@ -94,12 +97,24 @@ func handleError(message string, h *Handler) {
 
 // download a file and writes on the given writer
 func donwloadFile(url string, w io.Writer) ([]byte, error) {
-	resp, err := http.Get(url)
-	if err != nil {
-		return nil, fmt.Errorf("error downloading file from url %s, got error :%q", url, err)
+	var res *http.Response
+	var err error
+	if strings.Contains(url, "http") {
+		res, err = http.Get(url)
```

Comment:
Adicionar TODO para mudar para a biblioteca URLFetch, da AppEngine

Reply:
Aproveitando a brecha aqui, queria confirmar uma duvida: O urlfetch é uma obrigação para qualquer chamada para fora quando se está usando app engine?

Reply:
Não sei :)

Dei uma olhada e achei alguns exemplos:

- https://github.com/GoogleCloudPlatform/golang-samples/tree/master/appengine/go11x
- https://github.com/GoogleCloudPlatform/golang-samples/tree/master/appengine_flexible

Reply:
Pelo que entendi aqui (https://cloud.google.com/appengine/docs/standard/python/issue-requests?hl=pt-br#urlfetch) é isso mesmo, se o request for sair de dentro do app engine tem que usar esse negocio. Vamo ver na pratica!

Reply:
Também acho que precisa. Enfim .. estou entendendo que você vai testar e, caso não funcione, vai mudar. Tudo bem por mim.